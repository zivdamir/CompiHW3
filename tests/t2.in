int myfunc(int x) {
    x = x + 3;
    return x;
}

int return6(int x) {
    int y = 6;
    return y;
}

void main() {
    bool c = false;
    if (not c) {
        int x = 6;
        printi(return6(x));
    }
    else
        syntax error;
}