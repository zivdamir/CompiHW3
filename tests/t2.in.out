---end scope---
x INT -1
---end scope---
x INT -1
y INT 0
---end scope---
x INT 1
---end scope---
line 18: syntax error
